src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ScanCommandCallable.java

```diff
@@ -24,9 +24,11 @@
 import com.synopsys.integration.log.IntLogger;
 import com.synopsys.integration.util.IntEnvironmentVariables;
 import com.synopsys.integration.util.NameVersion;
+import com.synopsys.integration.util.OperatingSystemType;
 
 public class ScanCommandCallable implements Callable<ScanCommandOutput> {
     private static final List<String> DRY_RUN_FILES_TO_KEEP = Arrays.asList("data");
+    private static final int WINDOWS_CHARACTER_LIMIT = 32108;
```

Comment:
Mostly curious, how did you determine this number?

Reply:
Will answer in above reply.